cparse.nim

```diff
@@ -2172,6 +2172,11 @@ proc parseConstructor(p: var Parser, pragmas: PNode, isDestructor: bool;
     let body = compoundStatement(p)
     if pfKeepBodies in p.options.flags:
       result.sons[bodyPos] = body
+  of pxAsgn:
+    # '= default;' C++11 defaulted constructor
+    getTok(p)
+    eat(p, pxSymbol)
```

Comment:
Already merged and adapted. But in principle, sure, seems nice to have an overloaded 'eat'. ;-)